src/pilot/dracclient_raid.patch

```diff
@@ -18,7 +18,16 @@
  class PhysicalDisk(PhysicalDiskTuple):
  
      def __new__(cls, **kwargs):
-@@ -633,15 +637,17 @@
+@@ -232,6 +236,8 @@
+         size_b = self._get_virtual_disk_attr(drac_disk, 'SizeInBytes')
+         drac_status = self._get_virtual_disk_attr(drac_disk, 'PrimaryStatus')
+         drac_raid_status = self._get_virtual_disk_attr(drac_disk, 'RAIDStatus')
++        if not drac_raid_status:
```

Comment:
Hey Rachit,

I don't think this will work because the code will throw an exception if the RAIDStatus attribute is not present.  See:
https://github.com/openstack/python-dracclient/blob/master/dracclient/resources/raid.py#L204
and
https://github.com/openstack/python-dracclient/blob/091b5167258f521deba3594f93fc199182612c22/dracclient/utils.py#L95

Reply:
Hi Chris, 
I got your point. I did not test this changes. Actually i forget to mention that "i have not tested this changes" in description and also i was not aware of the exception. Apologies..!.
Currently i am blocked on issue of node getting stuck in clean wait while testing(experimental_upstream_ironic and master) merge changes. So i was not able to reach at that point where this method gets invoke.
But after resolving above changes, i will make sure to test this PR changes and will you know here."

Reply:
Thanks Rachit!